### ISSUE TITLE
fails to start

### ISSUE THREAD

**Issue by ccomly** [ukdtom/WebTools.bundle#528]
## Environment
- **PMS Platform(s) OS**: arch linux
- **PMS Version**: 1.14.1.5488-cc260c476 December 14, 2018
- **Name of Browser**: chrome

## Error shown

 File "/var/lib/plex/Plex Media Server/Plug-ins/WebTools.bundle/Contents/Code/webSrv.py", line 453, in start_tornado
    application = Application(httpHandlers, **settings)
  File "/opt/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/code/sandbox.py", line 19, in _apply
    return apply(f, args, kwargs)
  File "/opt/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/tornado/web.py", line 1012, in __init__
    if handlers: self.add_handlers(".*$", handlers)
  File "/opt/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/tornado/web.py", line 1043, in add_handlers
    spec = URLSpec(pattern, handler, kwargs)
  File "/opt/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/tornado/web.py", line 1464, in __init__
    self.regex = re.compile(pattern)
  File "/opt/plexmediaserver/Resources/Python/lib/python2.7/re.py", line 194, in compile
    return _compile(pattern, flags)
  File "/opt/plexmediaserver/Resources/Python/lib/python2.7/re.py", line 251, in _compile
    raise error, v # invalid expression
error: unbalanced parenthesis

- **Symptom**: 

## Steps to reproduce
1. 
fresh plex installl, install webtools, start plex, it sometimes opens the settings for webtools, sometimes not. restart plex and then always times out, and no connection can be made to :33400 or the other port. also mention that the ports are up at first when it semi functional. but doesnt load the login page
 
## Expected behaviour
1. 

## Actual behaviour
1. 

## Additional Info
[Logs2.tar.gz](https://github.com/ukdtom/WebTools.bundle/files/2716070/Logs2.tar.gz)

## Checklist
- [x] Listed environment
- [x] Included PMS log zip and WebTools log (THIS IS MANDATORY!)
- [ ] Included screenshots (where appropriate)
- [x] Checked that this isn't already an [issue](https://github.com/ukdtom/WebTools.bundle/issues)

**Comment by ukdtom** [comment]
Looks like a bad install, AFAICT.

Try first with a manual reset
https://github.com/ukdtom/WebTools.bundle/wiki/Reset#manually

And if that doesn't help, reapply

**Comment by ukdtom** [comment]
Ping

**Comment by ccomly** [comment]
resetting kinda worked but offered a new issue.

after first start of plex after reset, i can get into the config screen and the ports are up,
33400 33443. connecting to these ports only gets timed out. 

on webtools also suggests to open a new tab connecting to ip:port, the ip is not my ip tho and there is no such ip on my lan. suggests 192.168.1.15:33400 /33443. 

changing base url in config to 192.168.1.10 (the acual ip)  and hitting save never completes. and accessing configuration is no longer possible as it will load forever.

webtools config is no longer accessible until i do a fresh reset, where it will work once and then not until next reset.

restarting plex has no effect, config still loads forever, the ports are up but just time out.

**Comment by ccomly** [comment]
magically started to work, not sure why or what i did.
